draft-secure-credential-transfer.md

```diff
@@ -315,22 +315,21 @@ Value is defined as a combination of the following values: "R" - for read access
 Status: “200” (OK)
 
 ResponseBody:
+
 - urlLink (String, Required) - a full URL link to the mailbox including fully qualified domain name and mailbox Identifier.
+- isPushNotificationSupported (boolean, Required) - indicates whether push notification is supported or not. The device uses this field to decide whether it should listen on the push topic or do long-polling.
```

Comment:
The name seems a off, although it works. What do you think about something like `isPushNotificationTokenSupported` or `isPushNotificationTypeSupported`? And then the description would be:

> indicates whether the push notification token is supported or not. The device uses this field to decide whether it should listen on the push topic or do long-polling.

Reply:
I just generalized to isPushNotificationSupported instead of isPushNotificationTokenSupported or isPushNotificationTypeSupported.

case 1:  if we receive PushNotificationToken and if we don't support the notification type
case 2:  if we don't receive PushNotificationToken and if we support the notification type